extensions/schema-compare/src/schemaCompareMainWindow.ts

```diff
@@ -1135,7 +1149,11 @@ export class SchemaCompareMainWindow {
 				databaseName: '',
 				ownerUri: '',
 				packageFilePath: endpoint.packageFilePath,
-				connectionDetails: undefined
+				connectionDetails: undefined,
+				projectFilePath: '',
+				targetScripts: [],
```

Comment:
why are these being added when they aren't getting set since it isn't a sql project endpoint in this case?

Reply:
This isn't necessary, I will remove it in next iteration.